WalletWasabi.Fluent/Views/Wallets/Send/TransactionPreviewView.axaml

```diff
@@ -67,73 +67,78 @@
                   <DockPanel>
                     <TextBlock Text="Suggestions to improve your privacy" HorizontalAlignment="Center" DockPanel.Dock="Top" />
 
-                    <ListBox DockPanel.Dock="Top" Background="Transparent" VerticalAlignment="Center"
-                             HorizontalAlignment="Center"
-                             Items="{Binding Suggestions}" SelectedItem="{Binding SelectedSuggestion}">
-                      <i:Interaction.Behaviors>
-                        <behaviors:ListBoxPreviewBehavior PreviewItem="{Binding PreviewSuggestion, Mode=TwoWay}" />
-                      </i:Interaction.Behaviors>
-                      <ListBox.Styles>
-                        <Style Selector="ListBoxItem">
-                          <Setter Property="Cursor" Value="Hand" />
-                        </Style>
-                        <Style Selector="ListBoxItem /template/ ContentPresenter#PART_ContentPresenter">
-                          <Setter Property="Background" Value="Transparent" />
-                        </Style>
+                    <Panel>
```

Comment:
change it to a `StackPanel` with a `Horizontal` orientation, so the items in the list will be always on left and the spinner will be always on the left (as in the original solution).